packages/cli/src/services/McpPromptLoader.ts

```diff
@@ -123,7 +123,11 @@ export class McpPromptLoader implements ICommandLoader {
                 };
               }
 
-              if (!result.messages?.[0]?.content?.text) {
+              const content = result.messages
```

Comment:
We wont want to return `content` as a string here. Ultimately the the `submit_prompt` return type may need to be updated to take a `PartListUnion` type so that re-submission can take a complex data type. This would also then enable multi-part content to be returned from the prompts endpoint